diff --git a/ocamltest/ocaml_compilers.ml b/ocamltest/ocaml_compilers.ml
index a47c2ae6f..90d62b8b7 100644
--- a/ocamltest/ocaml_compilers.ml
+++ b/ocamltest/ocaml_compilers.ml
@@ -40,14 +40,20 @@ class compiler
   method target = target
 
   method program_variable =
+    Builtin_variables.program
+    (*
     if Ocaml_backends.is_native host
     then Builtin_variables.program2
     else Builtin_variables.program
+    *)
 
   method program_output_variable =
+    Some Builtin_variables.output
+    (*
     if Ocaml_backends.is_native host
     then None
     else Some Builtin_variables.output
+    *)
 
   method ! reference_file env prefix =
     let default = tool#reference_file env prefix in
diff --git a/ocamltest/ocaml_tests.ml b/ocamltest/ocaml_tests.ml
index b42172d37..6cf4f6718 100644
--- a/ocamltest/ocaml_tests.ml
+++ b/ocamltest/ocaml_tests.ml
@@ -22,10 +22,12 @@ open Ocaml_actions
 let bytecode =
   let opt_actions =
   [
+(*
     setup_ocamlc_opt_build_env;
     ocamlc_opt;
     check_ocamlc_opt_output;
     compare_bytecode_programs
+*)
   ] in
 {
   test_name = "bytecode";
@@ -43,14 +45,16 @@ let bytecode =
 let native =
   let opt_actions =
   [
+(*
     setup_ocamlopt_byte_build_env;
     ocamlopt_byte;
     check_ocamlopt_byte_output;
-    run;
-    check_program_output;
+*)
     setup_ocamlopt_opt_build_env;
     ocamlopt_opt;
     check_ocamlopt_opt_output;
+    run;
+    check_program_output;
   ] in
   {
     test_name = "native";
