diff --git a/ocamltest/ocaml_tests.ml b/ocamltest/ocaml_tests.ml
index 55eaea2fa..86431559c 100644
--- a/ocamltest/ocaml_tests.ml
+++ b/ocamltest/ocaml_tests.ml
@@ -22,9 +22,11 @@ open Ocaml_actions
 let bytecode =
   let opt_actions =
   [
+ (*
     setup_ocamlc_opt_build_env;
     ocamlc_opt;
     check_ocamlc_opt_output;
+ *)
   ] in
 {
   test_name = "bytecode";
@@ -47,9 +49,11 @@ let native =
     check_ocamlopt_byte_output;
     run;
     check_program_output;
+ (*
     setup_ocamlopt_opt_build_env;
     ocamlopt_opt;
     check_ocamlopt_opt_output;
+ *)
   ] in
   {
     test_name = "native";
